plugin-server/src/main/pluginsServer.ts

```diff
@@ -119,6 +119,12 @@ export async function startPluginsServer(
         process.exit(0)
     })
 
+    process.on('unhandledRejection', (error: Error) => {
```

Comment:
@mariusandra here I'm doing the same thing the workers do but also on the main thread. We void things here too and wouldn't want to crash if those throw